bundle/edu.gemini.spdb.rollover.servlet/src/main/scala/edu/gemini/rollover/servlet/RolloverServlet.scala

```diff
@@ -55,10 +55,12 @@ final class RolloverServlet(site: Site, odb: IDBDatabaseService, user: java.util
     <obs>
       <id>{ ro.id.toString }</id>
       <partner>{ ro.partner }</partner>
-      <target>
-        <ra>{ ro.target.coords.map(_.ra).getOrElse(0.0) }</ra>
-        <dec>{ ro.target.coords.map(_.dec).getOrElse(0.0) }</dec>
-      </target>
+      {ro.targets.map { t =>
+        <target>
+          <ra>{  t.coords.map(_.ra).getOrElse(0.0) }</ra>
+          <dec>{ t.coords.map(_.dec).getOrElse(0.0) }</dec>
+        </target>
+      }}
```

Comment:
Yes, verified that it will continue to work with multiple targets. ITAC just [grabs the first one](https://github.com/gemini-hlsw/itac/blob/7ce7fcd341c026c6ac9a37a5ce472e79acc267f4/phase1-data/src/main/java/edu/gemini/tac/persistence/rollover/RolloverObservation.java#L58).